mica_ilp.cpp

```diff
@@ -138,9 +138,9 @@ void init_ilp_one(){
 			cerr << "ERROR! Interval size is not a multiple of ILP buffer size. (" << interval_size << " vs " << ILP_BUFFER_SIZE << ")" << endl;
 			exit(-1);
 		}
-		stringstream filename;
-		filename << "ilp-win" << win_size << "_phases_int";
-		output_file_ilp_one.open(mkfilename(filename.str().c_str()), ios::out|ios::trunc);
+		char filename[100];
+        sprintf(filename, "ilp-win%d_phases_int", win_size);
```

Comment:
what's up with the funky indentation here?

Reply:
That's right. Sort of a Pythonish indentation.